rasa/core/policies/default_flows.yml

```diff
@@ -5,28 +5,12 @@ responses:
       metadata: 
         rephrase: True
         template: jinja
-
-  utter_ask_confirm_correction:
-    - text: "Do you want to update the {{ context.corrected_slots.keys()|join(', ') }}?"
-      buttons:
-        - payload: yes
-          title: Yes
-        - payload: no
-          title: No, please keep the previous information
-      metadata: 
-        rephrase: True
-        template: jinja
```

Comment:
how come this was removed? Because we don't ask anymore by default?

Reply:
yes - i think we discussed it multiple times in the chat to remove the confirmation.

i did add it as a custom pattern implementation for the test assistant and also in the docs as a possible modification.

Reply:
created an issue https://rasahq.atlassian.net/browse/ENG-503